data/json/monsters.json

```diff
@@ -926,7 +934,22 @@
     "armor_cut": 6,
     "vision_day": 50,
     "vision_night": 3,
-    "special_attacks": [ [ "LONGSWIPE", 8 ] ],
+    "special_attacks": [
+      [ "LONGSWIPE", 8 ],
+      {
+        "id": "scratch",
+        "damage_max_instance": [
+          {
+            "damage_type": "cut",
+            "amount": 13
+          },
+          {
+            "damage_type": "stab",
+            "amount": 8
```

Comment:
Problem with dual damage is that it will bounce off even light armor, but rip apart unarmored parts.
Would be better to combine it into one.